data/ecosystem/adopters.yaml

```diff
@@ -95,6 +95,11 @@
   reference: /blog/2023/end-user-q-and-a-02/
   referenceTitle: blog post
   contact: ''
+- name: VTEX
```

Comment:
to fix the spelling issue add VTEX to line 1 of this document (the cspell ignore list)